src/App.js

```diff
@@ -1,42 +1,27 @@
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
-import logo from "./logo.png";
+import { Login } from "./components/Auth/Login";
```

Comment:
Suggestion - Instead of exporting components from different folders and files, you could create an index.js file inside your components folder. Then from this folder, you could export all the components. 
This way, you'd have to import all components from a single file.

Reply:
Yes I did this @bharati-21 Thanks For this suggestion